exercises/camicia/instructions.md

```diff
@@ -0,0 +1,83 @@
+# Instructions
+
+In this exercise, you will simulate a game very similar to the classic card game **Camicia**.
+Your program will receive the initial configuration of two players' decks and must simulate the game until it ends (or detect that it will never end).
+
+## Rules
+
+- The deck is split between **two players**.
+  The player's cards are read from left to right, where the leftmost card is the top of the deck.
+- A round consists of both players playing at least one card/turn.
+- Players take a turn by placing the **top card** of their deck onto a central pile.
+- If the card is a **number card** (2-10), play simply passes to the other player.
+- If the card is a **payment card**, a penalty must be paid:
+  - `J` → opponent must pay 1 card
+  - `Q` → opponent must pay 2 cards
+  - `K` → opponent must pay 3 cards
+  - `A` → opponent must pay 4 cards
```

Comment:
```suggestion
  - J → opponent must pay 1 card
  - Q → opponent must pay 2 cards
  - K → opponent must pay 3 cards
  - A → opponent must pay 4 cards
```

Reply:
These could be bold or italicized but codeblocks should be reserves for code and/or data.